OpenEdXMobile/src/main/java/org/edx/mobile/http/HttpResponseStatusException.java

```diff
@@ -1,13 +1,37 @@
 package org.edx.mobile.http;
 
+import android.support.annotation.NonNull;
+
+import retrofit2.Response;
+
+/**
+ * Signals an HTTP status error.
+ */
 public class HttpResponseStatusException extends Exception {
```

Comment:
Might as well refactor this class's name to `HttpResponseException` cuz `Status` is limiting its scope.